rules/sinks/storages/amazonS3/python.yaml

```diff
@@ -7,6 +7,7 @@ sinks:
     patterns:
       - "(?i)(aws_cdk.*aws_s3).*"
       - "(?i)(aws_cdk.*aws_s3_assets).*"
+      - "(?i)(aws_cdk.*assets).*"
```

Comment:
@dbMundada The 4 regex provided here are overlapping, better to have independent match regex